packages/paste-core/components/side-panel/src/SidePanel.tsx

```diff
@@ -1,12 +1,41 @@
 import { animated, useTransition } from "@twilio-paste/animation-library";
-import { Box, safelySpreadBoxProps } from "@twilio-paste/box";
+import { Box, getCustomElementStyles, safelySpreadBoxProps } from "@twilio-paste/box";
 import type { BoxProps } from "@twilio-paste/box";
+import { ModalDialogPrimitiveContent, ModalDialogPrimitiveOverlay } from "@twilio-paste/modal-dialog-primitive";
+import { css, styled } from "@twilio-paste/styling-library";
+import { pasteBaseStyles, useTheme } from "@twilio-paste/theme";
 import { useMergeRefs, useWindowSize } from "@twilio-paste/utils";
 import * as React from "react";
 
 import { SidePanelContext } from "./SidePanelContext";
 import type { SidePanelProps } from "./types";
 
+const SidePanelMobileOverlay = animated(
+  // @ts-expect-error the styled div color prop from emotion is clashing with our color style prop in BoxProps
+  styled(ModalDialogPrimitiveOverlay)(
+    css({
+      backgroundColor: "colorBackgroundOverlay",
+      position: "fixed",
+      top: 0,
+      right: 0,
+      bottom: 0,
+      left: 0,
+      width: "100%",
+      zIndex: "zIndex80",
+    }),
+    /*
+     * import Paste Theme Based Styles due to portal positioning.
+     * reach portal is a sibling to the main app, so you are now
```

Comment:
super picky nit: is there a typo here?

```suggestion
     * react portal is a sibling to the main app, so you are now
```

Reply:
it's referring to the library we use for our modal primitive (https://reach.tech/dialog/)